chrome/scripts.js

```diff
@@ -0,0 +1,33 @@
+function remainingTime(targetDate) {
+    var currentDate = new Date().getTime();
+    var interval = targetDate - currentDate;
+
+    var days = Math.floor(interval / (1000 * 60 * 60 * 24));
+    var hours = Math.floor((interval % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+    var minutes = Math.floor((interval % (1000 * 60 * 60)) / (1000 * 60));
+    var seconds = Math.floor((interval % (1000 * 60)) / 1000);
+    return {
+        totalInterval: interval,
+        days: days,
+        hours: hours,
+        minutes: minutes,
+        seconds: seconds,
+    };
+}
+
+function setClock(date) {
+    var time = remainingTime(date);
+    document.getElementById("timer").innerHTML = time.days + "d " + time.hours + "h " + time.minutes + "m " + time.seconds + "s Left";
+
+    if (time.totalInterval <= 0) {
+        clearInterval(x);
+        document.getElementById("timer").innerHTML = "Contest has Started";
+    }
+}
+
+function runClock() {
```

Comment:
give this function the targetDate as its argument and we are golden!

Reply:
`function runClock(targetDate) {
    var targetDate = new Date("Nov 4, 2021").getTime();
    setClock(targetDate);
}` like this?

Reply:
No the function should take in its argument, the target time of the timer. 

like runClock(new Date("Nov 10, 2020"));